content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -56,8 +56,10 @@ different Kubernetes components.
 | `CRIContainerLogRotation` | `false` | Alpha | 1.10 | 1.10 |
 | `CRIContainerLogRotation` | `true` | Beta| 1.11 | |
 | `CSIBlockVolume` | `false` | Alpha | 1.11 | |
-| `CSIDriverRegistry` | `false` | Alpha | 1.12 | |
-| `CSINodeInfo` | `false` | Alpha | 1.12 | |
+| `CSIDriverRegistry` | `false` | Alpha | 1.12 | 1.13 |
+| `CSIDriverRegistry` | `beta` | Beta | 1.14 | |
```

Comment:
`beta` => `true` in second column

Reply:
Also both entries point to `CSIDriverRegistry`. I think the second one should point to `CSINodeInfo`?

Reply:
good catch. fixed